package.json

```diff
@@ -47,10 +47,9 @@
     "@storybook/blocks": "^7.0.22",
     "@storybook/react": "^7.0.22",
     "@storybook/react-vite": "^7.0.22",
-    "@storybook/testing-library": "^0.0.14-next.2",
```

Comment:
I removed the storybook version as it was clashing with the version we use for testing. Updated the user even library.